default/scripting/ship_hulls/ship_hulls.macros

```diff
@@ -171,4 +171,32 @@ AVERAGE_BASE_FUEL_REGEN
    [[REFUEL_MESSAGE(0.1001)]]
 '''
 
+GREAT_FUEL_HULL
+'''[[HULL_FUEL_EFFECTSGROUP(GREAT,Value * 2)]]
+'''
+
+GOOD_FUEL_HULL
+'''
+'''
+
+AVERAGE_FUEL_HULL
+'''[[HULL_FUEL_EFFECTSGROUP(AVERAGE,Value * 0.5)]]
```

Comment:
The "average" efficiency should probably correspond to a factor of 1.

Reply:
The problem here is that I would need to introduce a VERY_BAD level for the 30% and there is actually never a need for something better than GREAT.

Hmm... i could multiply all base fuel capacity by 0.5 and double the effects. Would that be a good idea?

Reply:
As large hulls have average fuel efficiency, rescaling everything so that average fuel is 100% means also that  for medium size hulls 0.5 fuel capacity means one jump.

Reply:
Implemented the rescaling.